Tasks/AppCenterTestV1/appcentertest.ts

```diff
@@ -209,7 +209,7 @@ async function setTestRunIdBuildPropertyAsync(testRunId: string) {
         let auth = token.length == 52 ? apim.getPersonalAccessTokenHandler(token) : apim.getBearerHandler(token);
         let vsts: apim.WebApi = new apim.WebApi(url, auth);
         let conn: lim.ConnectionData = await vsts.connect();
-        let buildApi = vsts.getBuildApi();
+        let buildApi = await vsts.getBuildApi();
```

Comment:
Why we use `await` here?

Reply:
This method returns a promise

![image](https://user-images.githubusercontent.com/93121155/174841674-ce5bf608-939e-4239-89e1-082f2a450d89.png)

Reply:
The link in the online editor leads to the old version of the type definition
![image](https://user-images.githubusercontent.com/93121155/174842779-3653c290-088e-4af8-b5b0-ad9ce4e1de4f.png)